src/energyplus/ForwardTranslator.cpp

```diff
@@ -108,9 +108,13 @@ namespace energyplus {
     m_excludeSQliteOutputReport = false;
     m_excludeHTMLOutputReport = false;
     m_excludeVariableDictionary = false;
+    m_excludeSpaceTranslation = false;
```

Comment:
Don't we want the default to be true for the moment? Or is the thinking that we handle this defaulting at the osw level?

Reply:
See conversation above. Basically we set it to false now while we work on the PR so we get the CI results (and so we don't have to tweak the OS-resources repo either) . We'll change when we feel PR is ready